docs/wiki/development/windows-provisioning.md

```diff
@@ -4,6 +4,8 @@ The bulk of the development environment provisioning logic is in the `tools\prov
 
 Generating the Visual Studio 2015 solution and building the **osquery** binaries is done via the `tools\make-win64-binaries.bat` batch script. 
 
+Please note that both above batch scripts needs  to be run through repo root. i.e their invocation is as their path appear above. 
```

Comment:
nit: There's an extra space between `needs` and `to`. Also some of the wording could possibly flow better? Maybe word it as follows and add emphasis to the 'please note' line?

**Note**: Both batch scripts above need to be run from the repo root, i.e their invocation should be exactly as they appear above. Read '**Build Process**' below for more details on provisioning.